MyProjectsCard Bootstrap CSS

rohitk18 opened the pull request:
Used Bootstrap CSS and removed table html.
Implemented for cards in MyProjects and MyGroups (uses same CSS classes)

PeterBreen commented on civictechprojects/static/css/partials/_MyProjectCard.scss:
```diff
@@ -16,14 +16,9 @@
-  margin: 20px auto;
-  padding: 16px;
-  width: 98%;
+  &.text-overflow-break {
```
This works - but to save a few lines of CSS how about using the built-in bootstrap utility class `text-break` to do the same thing? ([ref](https://getbootstrap.com/docs/4.4/utilities/text/#word-break))

rohitk18 replied:
Replaced with the built-in bootstrap utility class `text-break`

PeterBreen reviewed (changes requested):
This looks really good; I have one small requested change to a utility class, beyond that no other changes needed. Thanks for taking this issue! 👍

rohitk18 commented:
Replaced with the built-in bootstrap utility class `text-break` in project card and group card
Removed `text-overflow-break` from _MyProjectCard.scss

PeterBreen reviewed (approved):
👍 Perfect, thanks!
